percona/controllers/pgc/pgc.go

```diff
@@ -254,9 +315,13 @@ func (c *Controller) handleStatuses() error {
 	for _, n := range ns.Items {
 		perconaPGClusters, err := c.Client.CrunchydataV1().PerconaPGClusters(n.Name).List(ctx, metav1.ListOptions{})
 		if err != nil {
-			return errors.Wrap(err, "get percona clusters list")
+			return nil
```

Comment:
we've got an error, we need to log it at least

Reply:
https://github.com/percona/percona-postgresql-operator/pull/71/commits/a8cbb68cc0989a1d752f02faecd671470266675a